lifecycle_event.go

```diff
@@ -0,0 +1,24 @@
+package cyclist
+
+import "time"
+
+var (
+	standardFluxCapacitorTime, _ = time.Parse(time.RFC3339, "1955-11-05T11:05:55-09:00")
+)
+
+type lifecycleEvent struct {
+	Event     string    `json:"event"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+func newLifecycleEvent(event, ts string) *lifecycleEvent {
+	timestamp, err := time.Parse(time.RFC3339Nano, ts)
+	if err != nil {
+		timestamp = standardFluxCapacitorTime
```

Comment:
Is there any significance in this date? Looks like it's "way in the past" so it's always smaller than any date we expect, or is it just a "semi-random" date? Wondering if a `nil` or something like that would make more sense?